src/main/java/com/github/javafaker/service/FakeValuesService.java

```diff
@@ -31,6 +27,8 @@
     private final List<FakeValuesInterface> fakeValuesList;
     private final RandomService randomService;
 
+    private final List<Locale> locales;
```

Comment:
I would rename that variable to reflect the locale chain. As it could bring to confusion with the locale given in parameter. Something like `localesChain` or `localesChainFallback`?

Reply:
I like `localesChain`. :) And I thought, maybe `public List<Locale> getLocales()` change to package visibility?
With renaming will
```java
List<Locale> getLocalesChain()
```

Reply:
Yes, it could be an idea, but as I also have a different idea in mind, to manipulate the chain outside the package, I would still believe it is okay to have the getter public. But rename it anyway 👍

Reply:
Renamed.